[2021-12-15 04:57:56,277] {processor.py:163} INFO - Started process (PID=63875) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 04:57:56,277] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-15 04:57:56,278] {logging_mixin.py:109} INFO - [2021-12-15 04:57:56,278] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 04:57:56,296] {processor.py:632} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 04:57:56,301] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 04:57:56,308] {logging_mixin.py:109} INFO - [2021-12-15 04:57:56,308] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 04:57:56,319] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 04:57:56,323] {logging_mixin.py:109} INFO - [2021-12-15 04:57:56,323] {dag.py:2914} INFO - Setting next_dagrun for test_utils to None
[2021-12-15 04:57:56,334] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py took 0.060 seconds
[2021-12-15 04:58:30,641] {processor.py:163} INFO - Started process (PID=64182) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 04:58:30,642] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-15 04:58:30,642] {logging_mixin.py:109} INFO - [2021-12-15 04:58:30,642] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 04:58:30,663] {processor.py:632} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 04:58:30,669] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 04:58:30,676] {logging_mixin.py:109} INFO - [2021-12-15 04:58:30,675] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 04:58:30,686] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 04:58:30,691] {logging_mixin.py:109} INFO - [2021-12-15 04:58:30,691] {dag.py:2914} INFO - Setting next_dagrun for test_utils to None
[2021-12-15 04:58:30,702] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py took 0.064 seconds
[2021-12-15 05:04:26,903] {processor.py:163} INFO - Started process (PID=912) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:04:26,904] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-15 05:04:26,904] {logging_mixin.py:109} INFO - [2021-12-15 05:04:26,904] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:04:26,923] {processor.py:632} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:04:26,928] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:04:26,934] {logging_mixin.py:109} INFO - [2021-12-15 05:04:26,934] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:04:26,944] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:04:26,948] {logging_mixin.py:109} INFO - [2021-12-15 05:04:26,948] {dag.py:2914} INFO - Setting next_dagrun for test_utils to None
[2021-12-15 05:04:26,958] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py took 0.058 seconds
[2021-12-15 05:05:01,263] {processor.py:163} INFO - Started process (PID=1064) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:05:01,264] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-15 05:05:01,265] {logging_mixin.py:109} INFO - [2021-12-15 05:05:01,264] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:05:01,284] {processor.py:632} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:05:01,289] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:05:01,295] {logging_mixin.py:109} INFO - [2021-12-15 05:05:01,295] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:05:01,305] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:05:01,310] {logging_mixin.py:109} INFO - [2021-12-15 05:05:01,310] {dag.py:2914} INFO - Setting next_dagrun for test_utils to None
[2021-12-15 05:05:01,321] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py took 0.060 seconds
[2021-12-15 05:05:36,178] {processor.py:163} INFO - Started process (PID=1373) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:05:36,179] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-15 05:05:36,180] {logging_mixin.py:109} INFO - [2021-12-15 05:05:36,179] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:05:36,199] {processor.py:632} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:05:36,204] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:05:36,210] {logging_mixin.py:109} INFO - [2021-12-15 05:05:36,210] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:05:36,219] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:05:36,223] {logging_mixin.py:109} INFO - [2021-12-15 05:05:36,223] {dag.py:2914} INFO - Setting next_dagrun for test_utils to None
[2021-12-15 05:05:36,233] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py took 0.058 seconds
[2021-12-15 05:06:09,943] {processor.py:163} INFO - Started process (PID=1482) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:06:09,944] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-15 05:06:09,945] {logging_mixin.py:109} INFO - [2021-12-15 05:06:09,945] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:06:09,964] {processor.py:632} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:06:09,969] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:06:09,976] {logging_mixin.py:109} INFO - [2021-12-15 05:06:09,976] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:06:09,986] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:06:09,990] {logging_mixin.py:109} INFO - [2021-12-15 05:06:09,990] {dag.py:2914} INFO - Setting next_dagrun for test_utils to None
[2021-12-15 05:06:10,001] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py took 0.061 seconds
[2021-12-15 05:06:44,806] {processor.py:163} INFO - Started process (PID=1634) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:06:44,807] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-15 05:06:44,807] {logging_mixin.py:109} INFO - [2021-12-15 05:06:44,807] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:06:44,827] {processor.py:632} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:06:44,832] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:06:44,839] {logging_mixin.py:109} INFO - [2021-12-15 05:06:44,838] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:06:44,848] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:06:44,852] {logging_mixin.py:109} INFO - [2021-12-15 05:06:44,852] {dag.py:2914} INFO - Setting next_dagrun for test_utils to None
[2021-12-15 05:06:44,863] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py took 0.060 seconds
[2021-12-15 05:07:19,053] {processor.py:163} INFO - Started process (PID=1721) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:07:19,054] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-15 05:07:19,054] {logging_mixin.py:109} INFO - [2021-12-15 05:07:19,054] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:07:19,074] {processor.py:632} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:07:19,078] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:07:19,084] {logging_mixin.py:109} INFO - [2021-12-15 05:07:19,084] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:07:19,094] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:07:19,098] {logging_mixin.py:109} INFO - [2021-12-15 05:07:19,098] {dag.py:2914} INFO - Setting next_dagrun for test_utils to None
[2021-12-15 05:07:19,109] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py took 0.058 seconds
[2021-12-15 05:07:53,977] {processor.py:163} INFO - Started process (PID=1806) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:07:53,977] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-15 05:07:53,978] {logging_mixin.py:109} INFO - [2021-12-15 05:07:53,978] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:07:53,997] {processor.py:632} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:07:54,001] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:07:54,007] {logging_mixin.py:109} INFO - [2021-12-15 05:07:54,007] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:07:54,017] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:07:54,021] {logging_mixin.py:109} INFO - [2021-12-15 05:07:54,021] {dag.py:2914} INFO - Setting next_dagrun for test_utils to None
[2021-12-15 05:07:54,031] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py took 0.057 seconds
[2021-12-15 05:08:28,862] {processor.py:163} INFO - Started process (PID=1890) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:08:28,863] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-15 05:08:28,864] {logging_mixin.py:109} INFO - [2021-12-15 05:08:28,863] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:08:28,885] {processor.py:632} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:08:28,891] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:08:28,897] {logging_mixin.py:109} INFO - [2021-12-15 05:08:28,897] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:08:28,907] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:08:28,913] {logging_mixin.py:109} INFO - [2021-12-15 05:08:28,913] {dag.py:2914} INFO - Setting next_dagrun for test_utils to None
[2021-12-15 05:08:28,925] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py took 0.067 seconds
[2021-12-15 05:09:03,987] {processor.py:163} INFO - Started process (PID=2008) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:09:03,988] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-15 05:09:03,989] {logging_mixin.py:109} INFO - [2021-12-15 05:09:03,989] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:09:04,008] {processor.py:632} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:09:04,012] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:09:04,019] {logging_mixin.py:109} INFO - [2021-12-15 05:09:04,018] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:09:04,028] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:09:04,033] {logging_mixin.py:109} INFO - [2021-12-15 05:09:04,032] {dag.py:2914} INFO - Setting next_dagrun for test_utils to None
[2021-12-15 05:09:04,043] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py took 0.059 seconds
[2021-12-15 05:09:37,964] {processor.py:163} INFO - Started process (PID=2241) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:09:37,965] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-15 05:09:37,966] {logging_mixin.py:109} INFO - [2021-12-15 05:09:37,966] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:09:37,985] {processor.py:632} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:09:37,990] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:09:37,996] {logging_mixin.py:109} INFO - [2021-12-15 05:09:37,996] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:09:38,006] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:09:38,011] {logging_mixin.py:109} INFO - [2021-12-15 05:09:38,011] {dag.py:2914} INFO - Setting next_dagrun for test_utils to None
[2021-12-15 05:09:38,022] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py took 0.060 seconds
[2021-12-15 05:10:15,037] {processor.py:163} INFO - Started process (PID=2403) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:10:15,039] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-15 05:10:15,040] {logging_mixin.py:109} INFO - [2021-12-15 05:10:15,039] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:10:15,060] {processor.py:632} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:10:15,067] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:10:15,074] {logging_mixin.py:109} INFO - [2021-12-15 05:10:15,074] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:10:15,083] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:10:15,088] {logging_mixin.py:109} INFO - [2021-12-15 05:10:15,088] {dag.py:2914} INFO - Setting next_dagrun for test_utils to None
[2021-12-15 05:10:15,098] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py took 0.064 seconds
[2021-12-15 05:10:52,365] {processor.py:163} INFO - Started process (PID=2670) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:10:52,366] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-15 05:10:52,366] {logging_mixin.py:109} INFO - [2021-12-15 05:10:52,366] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:10:52,385] {processor.py:632} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:10:52,389] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:10:52,395] {logging_mixin.py:109} INFO - [2021-12-15 05:10:52,395] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:10:52,405] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:10:52,409] {logging_mixin.py:109} INFO - [2021-12-15 05:10:52,409] {dag.py:2914} INFO - Setting next_dagrun for test_utils to None
[2021-12-15 05:10:52,420] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py took 0.058 seconds
[2021-12-15 05:11:30,341] {processor.py:163} INFO - Started process (PID=2921) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:11:30,342] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-15 05:11:30,342] {logging_mixin.py:109} INFO - [2021-12-15 05:11:30,342] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:11:30,362] {processor.py:632} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:11:30,367] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:11:30,373] {logging_mixin.py:109} INFO - [2021-12-15 05:11:30,373] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:11:30,383] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:11:30,387] {logging_mixin.py:109} INFO - [2021-12-15 05:11:30,387] {dag.py:2914} INFO - Setting next_dagrun for test_utils to None
[2021-12-15 05:11:30,398] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py took 0.060 seconds
[2021-12-15 05:12:07,357] {processor.py:163} INFO - Started process (PID=3041) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:12:07,359] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-15 05:12:07,359] {logging_mixin.py:109} INFO - [2021-12-15 05:12:07,359] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:12:07,378] {processor.py:632} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:12:07,382] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:12:07,388] {logging_mixin.py:109} INFO - [2021-12-15 05:12:07,388] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:12:07,398] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:12:07,403] {logging_mixin.py:109} INFO - [2021-12-15 05:12:07,403] {dag.py:2914} INFO - Setting next_dagrun for test_utils to None
[2021-12-15 05:12:07,413] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py took 0.059 seconds
[2021-12-15 05:12:41,970] {processor.py:163} INFO - Started process (PID=3199) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:12:41,971] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-15 05:12:41,971] {logging_mixin.py:109} INFO - [2021-12-15 05:12:41,971] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:12:41,991] {processor.py:632} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:12:41,996] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:12:42,002] {logging_mixin.py:109} INFO - [2021-12-15 05:12:42,002] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:12:42,012] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:12:42,016] {logging_mixin.py:109} INFO - [2021-12-15 05:12:42,016] {dag.py:2914} INFO - Setting next_dagrun for test_utils to None
[2021-12-15 05:12:42,030] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py took 0.063 seconds
[2021-12-15 05:13:15,727] {processor.py:163} INFO - Started process (PID=3317) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:13:15,729] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-15 05:13:15,730] {logging_mixin.py:109} INFO - [2021-12-15 05:13:15,730] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:13:15,751] {processor.py:632} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:13:15,757] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:13:15,763] {logging_mixin.py:109} INFO - [2021-12-15 05:13:15,763] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:13:15,774] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:13:15,779] {logging_mixin.py:109} INFO - [2021-12-15 05:13:15,779] {dag.py:2914} INFO - Setting next_dagrun for test_utils to None
[2021-12-15 05:13:15,789] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py took 0.065 seconds
[2021-12-15 05:13:48,946] {processor.py:163} INFO - Started process (PID=3446) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:13:48,947] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-15 05:13:48,947] {logging_mixin.py:109} INFO - [2021-12-15 05:13:48,947] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:13:48,966] {processor.py:632} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:13:48,970] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:13:48,976] {logging_mixin.py:109} INFO - [2021-12-15 05:13:48,976] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:13:48,986] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:13:48,991] {logging_mixin.py:109} INFO - [2021-12-15 05:13:48,991] {dag.py:2914} INFO - Setting next_dagrun for test_utils to None
[2021-12-15 05:13:49,001] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py took 0.058 seconds
[2021-12-15 05:14:27,558] {processor.py:163} INFO - Started process (PID=3662) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:14:27,559] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py for tasks to queue
[2021-12-15 05:14:27,559] {logging_mixin.py:109} INFO - [2021-12-15 05:14:27,559] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:14:27,578] {processor.py:632} INFO - DAG(s) dict_keys(['test_utils']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py
[2021-12-15 05:14:27,583] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:14:27,589] {logging_mixin.py:109} INFO - [2021-12-15 05:14:27,589] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:14:27,599] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:14:27,603] {logging_mixin.py:109} INFO - [2021-12-15 05:14:27,603] {dag.py:2914} INFO - Setting next_dagrun for test_utils to None
[2021-12-15 05:14:27,613] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/test_utils.py took 0.059 seconds
